DocumentDataAPI/DocumentDataAPITests/Data/Repositories/NpgDocumentRepositoryIntegrationTests.cs

```diff
@@ -19,7 +18,45 @@ public NpgDocumentRepositoryIntegrationTests()
         _logger = new Logger<NpgDocumentRepository>(new NullLoggerFactory());
         TestHelper.DeployDatabaseWithTestData();
     }
+    
+    [Fact]
+    public void Get_ReturnsCorrectRows()
+    {
+        // Arrange
+        NpgDocumentRepository repository = new(_connectionFactory, _logger);
```

Comment:
Det er muligt at have denne statement eksekvere i constructoren af klassen, så behøver i ikke at kalde den for hver test. Se for eksempel tests for wordratio